bcda/models/models.go

```diff
@@ -52,8 +53,9 @@ type JobKey struct {
 type ACO struct {
 	gorm.Model
 	UUID     uuid.UUID `gorm:"primary_key; type:char(36)" json:"uuid"` // uuid
-	Name     string    `json:"name"`                                   // name
-	ClientID string    `json:"client_id"`                              // software client id
+	CMSID    string    `gorm:"type:char(5)" json:"cms_id"`
```

Comment:
can/should we enforce uniqueness here?  What would happen if we had 2 acos with ID of A1234?